testsuite_overview.md

```diff
@@ -0,0 +1,310 @@
+Rest.li Client Test Suite
+=========================
+
+This directory contains a language-independent Rest.li client test suite.
+It is designed to be shared across languages, and provides test data and testing guidelines that can be followed when 
+testing a particular language implementation of the Rest.li client.
+
+This project also contains a Java TestNG suite that uses the test suite's data and test guidelines. 
+This example demonstrates how to use the test suite for a specific language implementation of the Rest.li client. 
```

Comment:
"a specific language binding implementation of Rest.li client." No need to have "the" in front of every occurrence of Rest.li client.